packages/ui/src/components/Navigation/components/CategoriesDropdown.tsx

```diff
@@ -0,0 +1,91 @@
+import { ChevronDownIcon } from '@heroicons/react/24/solid';
+import type { Category, Culture } from '@prezly/sdk';
+import { getCategoryUrl, getLocalizedCategoryData, LocaleObject } from '@prezly/theme-kit-core';
+import Link from 'next/link';
+import { useState } from 'react';
+import { twMerge } from 'tailwind-merge';
+
+import { Button } from '@/components/Button';
+import { Dropdown } from '@/components/Dropdown';
+import { useDevice } from '@/hooks';
+
+export interface Props {
+    categories: Category[];
+    locale: Culture['code'];
+}
+
+export function CategoriesDropdown({ categories, locale }: Props) {
```

Comment:
IMO, the Categories on desktop look real bad :(
<img width="2053" alt="Screenshot 2023-11-02 at 11 54 50" src="https://github.com/prezly/theme-kit-js/assets/1641218/730e74be-3cee-483c-892f-c2253e34a329">

Could you advise with Al/Hugo on how this can be improved?

Reply:
Updated